src/index.js

```diff
@@ -23,4 +46,136 @@ export default class OsmRequest {
   get endpoint() {
     return this._options.endpoint;
   }
+
+  /**
+   * Create a shiny new OSM node element, in a geoJSON format
+   * @param {number} lat
+   * @param {number} lon
+   * @param {[object]} [properties] Optional, initial properties
+   * @return {object}
+   */
+  createNodeElement(lat, lon, properties = {}) {
+    return {
+      type: 'Feature',
+      properties: { ...properties },
+      geometry: {
+        type: 'Point',
+        coordinates: [lat, lon]
+      }
+    };
+  }
+
+  /**
+   * Fetch an OSM element by its ID
+   * @param {string} osmId Eg: node/12345
+   * @return {Promise}
+   */
+  fetchElement(osmId) {
+    return fetchElementRequest(this.endpoint, osmId);
+  }
+
+  /**
+   * Add or replace a property in a given element
```

Comment:
why `property` ? This looks like a `tag` to me :yum:

Reply:
All the under the hood manipulations are done in a GeoJson format. Hence the `property`, GeoJson does not handle `tags`.

Reply:
Ok I did not understand that.

Not every OSM element can be easily converted to geojson: how do you handle this kind of relation https://www.openstreetmap.org/relation/7921012 ?

Reply:
Maybe we could introduce a geojson option : by default, we can provide a simple XML to JSON conversion of OSM API response, so that way we are able to handle every kind of objects retrieved (even relations only having relation as child). If option is set, we try to convert as much as possible into GeoJSON.